### ISSUE TITLE
Invalid Assert statement in SmtpMailService.java

### ISSUE THREAD

**Issue by georcon** [zandero/mail#2]
https://github.com/zandero/mail/blob/106f525def493780c8c68e6347d4941fd560faef/src/main/java/com/zandero/mail/service/smtp/SmtpMailService.java#L35

This can't be correct - it seems it only allows portnumbers which are negative or higher than 9999? What about good old 25?

I would have expected:

```
Assert.isFalse(port < 0 || port > 9999, "Invalid SMTP port given: " + port);
```
OR
```
Assert.isTrue(port > 0 && port < 10000, "Invalid SMTP port given: " + port);
```

**Comment by drejc** [comment]
Nicely spotted ... sure it is a mistake ... fix was published as version: 1.1.1 ...

**Comment by georcon** [comment]
@drejc You're the best! Thank you. Tested both original version and the 1.1.1 - no issues so far.
